internal/ui/termstatus/status.go

```diff
@@ -15,6 +15,13 @@ import (
 	"golang.org/x/text/width"
 )
 
+type TerminalWriter interface {
```

Comment:
Let's move this struct to `internal/ui/terminal.go` (or similar). This will finally allow us to stop importing `ui/termstatus` in `internal/ui/message.go`.